src/deriv_api/DerivAPIBasic.js

```diff
@@ -185,11 +192,23 @@ export default class DerivAPIBasic extends DerivAPICalls {
         return this.subscription_manager.forgetAll(...types);
     }
 
+    keepAlivePing() {
+        this.ping({ ping: 1 });
+        this.reconnect_timeout = setTimeout(this.reconnect.bind(this), 5000);
+    }
+
+    pong() {
+        clearTimeout(this.reconnect_timeout);
```

Comment:
Will be better to clear the value after cllear Timeout ? 
```suggestion
	if(this.reconnect_timeout){
            clearTimeout(this.reconnect_timeout);
            this.reconnect_timeout = false
        }
```